lib/resources/example_json/r4_example_binary.rb

```diff
@@ -0,0 +1,13 @@
+module Cerner
+  module Resources
+    R4_BINARY_ENTRY = {
+      resourceType: "Binary",
+      id: "XR-11707283",
+      meta: {
+          "lastUpdated": "2019-12-16T18:50:47.287+00:00"
+      },
+      contentType: "application/pdf",
+      data: "JVBERi0xLjQKJeLjz9MKMSAwIG9iago8PC9FbmNvZGlu <-- snipped for brevity -->"
```

Comment:
if you intend to do an html-style comment inside the data, it would be 
```
<!-- snipped for brevity -->
```

since pdf data isn't html, we probably don't need to do an html-style comment

```
      "data": "JVBERi0xLjQKJeLjz9MKMSAwIG9iago8PC9FbmNvZGlu... <snipped for brevity>"
```

Reply:
https://github.com/cerner/fhir.cerner.com/commit/70b171a1701f4afd9b85d8b10f6a8a609678eb5f#diff-9b4677563bd83e4f77cac2bb11986d7dL1

Reply:
I would rather not have the html-style comment

Reply:
https://github.com/cerner/fhir.cerner.com/commit/b059a8899514f91baa449669f35f9889fb89d5bf